### PR TITLE
Opt-out from gateway redirects done by IPFS Companion

### PR DESCRIPTION
**Pull request by lidel**
[Our browser extension](https://github.com/ipfs/ipfs-companion) redirects request to gateways, which breaks this status page. Additionally, due to a [workaround](https://github.com/ipfs-shipyard/ipfs-companion/pull/494) for [CORS bug in Firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=1450965)  it  produces [different results under Chrome and Firefox](https://user-images.githubusercontent.com/157609/41968786-afd51498-7a05-11e8-9941-130034ceb29f.png).

**TL;DR** we don't want browser extension to touch requests made by public-gateway-checker
This PR  adds an explicit opt-out hint that ensures test requests reach the right gateway.

More info about opt-out URL hint: https://github.com/ipfs-shipyard/ipfs-companion/pull/505 
(to be released with next version of browser extension)

cc @VictorBjelkholm

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -27,7 +27,9 @@ function checkGateways (gateways) {
   let checked = 0
   gateways.forEach((gateway) => {
     const gatewayAndHash = gateway.replace(':hash', hashToTest)
-    fetch(gatewayAndHash)
+    // opt-out from gateway redirects done by browser extension
+    const testUrl = gatewayAndHash + '#x-ipfs-companion-no-redirect'
```
Comment (lidel) [inline app.js]: @VictorBjelkholm  FYI this feature shipped to Beta in [ipfs-companion/releases/tag/v2.4.0.10120](https://github.com/ipfs-shipyard/ipfs-companion/releases/tag/v2.4.0.10120)

**Comment by lidel** [comment]
FYSA support for `x-ipfs-companion-no-redirect` shipped with [v2.4.1](https://github.com/ipfs-shipyard/ipfs-companion/releases/tag/v2.4.1) of our browser extension.

**Comment by VictorBjelkholm** [comment]
Thanks!
